libr/io/p/io_debug.c

```diff
@@ -96,8 +96,10 @@ static int setup_tokens() {
 }
 
 static int fork_and_ptraceme(RIO *io, int bits, const char *cmd) {
-	PROCESS_INFORMATION pi;
+	PROCESS_INFORMATION pi = { 0 };
```

Comment:
I'm not sure this actually zeroes the whole struct. Doesn't it just set the first member to 0?